examples/playground/src/index.tsx

```diff
@@ -20,6 +20,7 @@ provideCreateRoot(createRoot);
 
 const rootDOMNode = document.createElement("div");
 rootDOMNode.className = "root";
+rootDOMNode.style.height = "100%";
```

Comment:
Small change that puts dashboard loader into the centre of the Playground from its location at the top of the page. This bothered me for a long time 😅